conda/cli/common.py

```diff
@@ -77,6 +77,15 @@ def ensure_name_or_prefix(args, command):
         raise CondaValueError('either -n NAME or -p PREFIX option required,\n'
                               'try "conda %s -h" for more details' % command)
 
+def is_active_prefix(prefix):
+    """
+    Determines whether the args we pass in are pointing to the active prefix.
+    Can be used a validation step to make sure operations are not being
+    performed on the active prefix.
+    """
+    active_prefix = context.active_prefix
+    return paths_equal(prefix, active_prefix) or paths_equal(prefix, env_name(active_prefix))
```

Comment:
FYI, the reason I use the `active_prefix` variable here is because otherwise the linter repeatedly tries to re-format this bit of code to look like:
```
    return paths_equal(prefix, context.active_prefix) or paths_equal(
        prefix, env_name(context.active_prefix)
    )
```
...which I didn't think was super readable. This happened no matter how I reformatted the line; even something like the below didn't escape the linter's reformatting:
```
    return paths_equal(prefix, context.active_prefix) \
    or paths_equal(prefix, env_name(context.active_prefix))
```

Reply:
Extra parentheses should do the trick:

```suggestion
    return (
        paths_equal(prefix, context.active_prefix)
        or paths_equal(prefix, env_name(context.active_prefix))
    )
```

Reply:
`darker` is still insisting on this formatting:
```
    return paths_equal(prefix, context.active_prefix) or paths_equal(
        prefix, env_name(context.active_prefix)
    )
```

Is there any way to bypass this in `darker` without changing default checks for everything else?

Reply:
Ugh, black 🤦‍♂️

Adding comments like this will preserve the formatting, but the comments I came up with feel unnecessary

```python
    return (
        # check if the prefix is the active env path
        paths_equal(prefix, context.active_prefix)
        # check if the prefix is the active env name
        or prefix == env_name(context.active_prefix)
    )
```

Can also explicitly disable formatting, but personally, I don't think that's worth it, at this point, we might as well leave it with the `active_prefix` variable or just let black do its thing and leave it a little weird 🤷‍♂️

```python
    # fmt: off
    return (
        paths_equal(prefix, context.active_prefix)
        or prefix == env_name(context.active_prefix)
    )
    # fmt: on
```

Also, not sure we want to use `paths_equal` for the second condition?